alerter.c

```diff
@@ -1,31 +1,84 @@
 #include <stdio.h>
 #include <assert.h>
 
+
+#define PRODUCTION_ENV  1
+#define TESTCODE_ENV         2
+
 int alertFailureCount = 0;
+int Current_Env = PRODUCTION_ENV;
+
+/*Conversion from farenheit to celcius*/
+float convertfarenheittocelcius(float farenheit)
+{
+    float celcius = (farenheit - 32) * 5 / 9;
+    return celcius;
+}
 
 int networkAlertStub(float celcius) {

```

Comment:
production code and test code both are in the same file. Code needs rearrangement.